auto-self-nominate/README.md

```diff
@@ -0,0 +1,19 @@
+### Auto self nominate
+
+Automatically send nominate transaction.
+
+### Environment variables
+
+| Key                             | Description                                                                              | Example values                              | Default value |
+| ------------------------------- | ---------------------------------------------------------------------------------------- | ------------------------------------------- | ------------- |
+| NETWORK_ID                      | Network ID of connected CodeChain                                                        | cc, bc, tc                                  | null          |
+| RPC_URL                         | CodeChain HTTP JSON RPC URL                                                              | http://localhost:8080                       | null          |
+| NEED_NOMINATION_UNDER_TERM_LEFT | Send nomination when nomination will be ended after NEED_NOMINATION_UNDER_TERM_LEFT term | 2                                           | 2             |
+| ACCOUNT_ADDRESS                 | Platform address of sender                                                               | bccqy7aeyc0qryehxgtx9y4vthhg7u95rad2ymlya7j | null          |
+| PASSPHRASE                      | A passphrase that is used to encrypt the ACCOUNT_ADDRESS                                 | xxxx                                        | null          |
+| METADATA                        | A metadata will be writen in a nomination transaction                                    | Some advertisement text                     | null          |
```

Comment:
I think we don't need 'A' in front of metadata.

a nomination transaction -> nomination transactions